pkg/controller/clusteringress/resources/route.go

```diff
@@ -14,6 +14,10 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+const (
+	routeTimeout = "haproxy.router.openshift.io/timeout"
```

Comment:
nit

```suggestion
	timeoutAnnotation = "haproxy.router.openshift.io/timeout"
```

Reply:
Changed routeTimeout to timeoutAnnotation